mathlibtools/leanproject.py

```diff
@@ -204,7 +179,17 @@ def get_project(name: str, new_branch: bool, directory: str = '') -> None:
         LeanProject.from_git_url(url, directory, branch, new_branch,
                                  cache_url, force_download)
     except GitCommandError as err:
-        handle_exception(err, 'Git command failed')
+        # if full url is provided, do not retry with HTTPS
+        if ('Permission denied' in err.stderr) and not is_url:
```

Comment:
This is typical USA/UK programming... What about users whose locale is not English?

Reply:
guilty as charged – I don't think the git python API provides a sane way to parse `GitCommandError`, so I will just remove this. The worst that will happen is that if a project name is provided that does not exist (e.g. `leanproject get berfectoid`), it will be retried via HTTPS despite failing because it does not exist. Regardless, the user will see a `Repository not found` error after that.